### ISSUE TITLE
Please include instructions on how to find out Which Version of Windows 10 we have. 

### ISSUE THREAD

**Issue by SherylHohman** [MicrosoftDocs/Virtualization-Documentation#1126]
You tell us which versions of Microsoft are required...
But not how to find out which version we are currently running!
Your OS is not easy to navigate, and has changed drastically over a lifetime of use.
This is not a field that shows up anywhere, or something that we access everyday, thus most users have not committed the location of this piece of information to memory.   
I looked in Programs and Features, then chose to look at Microsoft updates, as well as a few other places. 
Excellent documentation would include a link to any references you make that user might be unfamiliar with, or it would embed the info directly in the instructions. 
Please update to make your docs more user friendly.  
User friendly docs makes Windows more user friendly.  Friendly makes more friends and users.

In this case I can do another internet search and will easily find an answer.  But a couple characters in the doc could save thousands of additional GOOGLE searches. And many people from "eh, forget it" drop off.
And quite frankly many of your docs have useless links, and searching for the required information is a tedious, time consuming, and unfruitful task.  So while this particular link may seem minor, it is symbolic of the terrible documentation standards you endemically employ.
Hopefully, you are on a path to improve, so here is an easy win. And a baseline example that will help multitudes of users.

https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 10344c5d-274e-de75-ecb6-11d53a377986
* Version Independent ID: 4cea53b8-1864-f12b-f0d9-f208743475c8
* Content: [Create a Virtual Machine with Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine)
* Content Source: [virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine.md](https://github.com/Microsoft/Virtualization-Documentation/blob/master/virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine.md)
* Product: **windows-10-hyperv**
* GitHub Login: @scooley

**Comment by tusharchandwani** [comment]
https://support.microsoft.com/en-gb/help/13443/windows-which-version-am-i-running

**Comment by mikerodionov** [comment]
Yep it would come in handy including link on how to get windows version in note under the list and/or phrase "Execute winver command to confirm your Windows version"

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
